app/helpers/displayHelpers.php

```diff
@@ -820,12 +822,13 @@ function caEditorFindResultNavigation($po_request, $po_instance, $po_result_cont
 				}
 				TooltipManager::add(".prev.record", "Previous"); 
 			} else {
+				// DO something to load previous page of results
```

Comment:
Will need to figure out if there is a previous page (current page > 1?), if so display link, on click will need to load the previous result page in to context? and then find the last result on that page?

we wont be able to do the permission checking here until  we know what that result is...

Reply:
@kehh Alternatively we could show the "true" count and position, but only allow you to page through the current page of results? (effectively current functionality as of ef8f5217d824fc8af8c1773a54a0ef9b9829bd71)

Reply:
Yes let's just go for this and we can always implement it in the future if required.